### PR TITLE
load addresses based on netId

### PR DESCRIPTION
**Pull request by rstormsf**

### CODE REVIEW

**Comment by igorbarinov** [comment]
@rstormsf Let's follow PR police we have in Wizard in other DApps https://github.com/poanetwork/ico-wizard/wiki/Pull-Request-Policy

Quote:
```
- (Mandatory) Description
   a human-readable description of changes
   a human-readable description of the purpose of the PR
- (Mandatory) What is it: (Fix), (Feature), or (Refactor) in Title, e.g., "(Fix) price of 1 token in Wei > 18 decimals"
- (Mandatory) Developers: Each completed PR should be updated in the Wiki Documentation.
(Recommended) Each PR should have one commit message and therefore should only contain one specific fix or feature. Otherwise, multiple PRs should be made
- (Optional) Any additional concerns or comments
```

File: src/contracts/addresses.js [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+        POA_ADDRESS: '0x8bf38d4764929064f2d4d3a56520a76ab3df415b',
+        MOC: '0xCf260eA317555637C55F70e55dbA8D5ad8414Cb0'
+      }
+    default:
+      return {
+        METADATA_ADDRESS: '0xcBB2912666c7e8023B7ec78B6842702eB26336aC',
+        KEYS_MANAGER_ADDRESS: '0xfc90125492e58dbfe80c0bfb6a2a759c4f703ca8',
+        POA_ADDRESS: '0x8bf38d4764929064f2d4d3a56520a76ab3df415b',
+        MOC: '0xCf260eA317555637C55F70e55dbA8D5ad8414Cb0'
+      }
```
Comment (vbaranov) [inline src/contracts/addresses.js]: A small suggestion here: core addresses are written twice. Let's create const:
```
const CORE_ADDRESSES = {
    METADATA_ADDRESS: '0xcBB2912666c7e8023B7ec78B6842702eB26336aC',
    KEYS_MANAGER_ADDRESS: '0xfc90125492e58dbfe80c0bfb6a2a759c4f703ca8',
    POA_ADDRESS: '0x8bf38d4764929064f2d4d3a56520a76ab3df415b',
    MOC: '0xCf260eA317555637C55F70e55dbA8D5ad8414Cb0'
 }
```
to use it as a default and for core network in switch. Other updates look good for me.

**Comment by igorbarinov** [comment]
Summarized out PR Policy for DApps in RFC (new format of sharing common policies)
Please comment https://github.com/poanetwork/RFC/issues/3
@vbaranov @rstormsf 

Btw, @rstormsf let's create an RFC about `core/sokol` branches modification and policy for all our DApps

File: src/contracts/addresses.js (hunk 1 above)
Comment (rstormsf) [reply src/contracts/addresses.js]: 👍  will do

**Pull request merged**